hemi/electrs/conn.go

```diff
@@ -86,6 +93,14 @@ func (c *clientConn) call(ctx context.Context, method string, params, result any
 	if err != nil {
 		return fmt.Errorf("create request: %w", err)
 	}
+
+	// Check context again before writing
+	select {
+	case <-ctx.Done():
+		return ctx.Err()
+	default:
+	}
+
```

Comment:
This is the first thing that happens in the underlying code and thus not needed.